zuul-ci/run-tests.yaml

```diff
@@ -0,0 +1,9 @@
+- hosts: all
+  roles:
+    - repo-setup
+  tasks:
```

Comment:
These tasks are there just to show how to access depends's artifacts rpms.
We can start to iterate on that playbook to implement the container build.